Running under Windows

Can this be run under Windows or Cygwin?

```powershell
PS C:\Users\MyUser> go get -u github.com/mumoshu/variant
PS C:\Users\MyUser> variant version
panic: open .C:\Users\MyUser\go\bin\variant.exeenv: The filename, directory name, or volume label syntax is incorrect.

goroutine 1 [running]:
github.com/mumoshu/variant/pkg/run.RunTaskDef(0xc00008c030, 0x2a, 0xc0000b4b00, 0xc00005e410, 0x1, 0x1)
        C:/Users/MyUser/go/src/github.com/mumoshu/variant/pkg/run/run.go:121 +0x18d0
github.com/mumoshu/variant/pkg/run.Dev()
        C:/Users/MyUser/go/src/github.com/mumoshu/variant/pkg/run/run.go:95 +0x31f
main.main()
        C:/Users/MyUser/go/src/github.com/mumoshu/variant/main.go:8 +0x27
```

--- a/pkg/application.go
+++ b/pkg/application.go
@@ -3,7 +3,7 @@ package variant
 import (
 	"fmt"
 	"os"
-	"path"
+	"path/filepath"
 	"strings"
 
 	bunyan "github.com/mumoshu/logrus-bunyan-formatter"
@@ -51,7 +51,7 @@ func (p Application) UpdateLoggingConfiguration() error {
 		p.Log.SetOutput(os.Stderr)
 	}
 
-	commandName := path.Base(os.Args[0])
+	commandName := filepath.Base(os.Args[0])
 	if p.Output == "bunyan" {
 		p.Log.SetFormatter(&bunyan.Formatter{Name: commandName})
 	} else if p.Output == "json" {

--- a/pkg/load/load.go
+++ b/pkg/load/load.go
@@ -3,11 +3,11 @@ package load
 import (
 	"github.com/mumoshu/variant/pkg"
 	"io/ioutil"
-	"path"
+	"path/filepath"
 )
 
 func File(cmdPath string) (*variant.TaskDef, error) {
-	cmdName := path.Base(cmdPath)
+	cmdName := filepath.Base(cmdPath)
 
 	yaml, err := ioutil.ReadFile(cmdPath)
 	if err != nil {

--- a/pkg/run/run.go
+++ b/pkg/run/run.go
@@ -15,7 +15,7 @@ import (
 	"github.com/mumoshu/variant/pkg/util/envutil"
 	"github.com/mumoshu/variant/pkg/util/fileutil"
 	"github.com/spf13/cobra"
-	"path"
+	"path/filepath"
 )
 
 func init() {
@@ -60,10 +60,10 @@ func Dev() {
 		varfile = os.Args[1]
 		args = os.Args[2:]
 		cmdPath = varfile
-		cmdName = path.Base(cmdPath)
+		cmdName = filepath.Base(cmdPath)
 	} else {
 		cmdPath = os.Args[0]
-		cmdName = path.Base(cmdPath)
+		cmdName = filepath.Base(cmdPath)
 		varfile = fmt.Sprintf("%s.definition.yaml", cmdName)
 		args = os.Args[1:]
 	}
@@ -114,7 +114,7 @@ func YAML(yaml string) {
 		panic(errors.Trace(err))
 	}
 
-	taskDef.Name = path.Base(cmdPath)
+	taskDef.Name = filepath.Base(cmdPath)
 
 	Def(taskDef, variant.Opts{
 		CommandPath: cmdPath,

